pkg/loader/compose/compose.go

```diff
@@ -358,6 +358,11 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 			}
 		}
 
+		if len(serviceConfig.Port) == 0 && composeServiceConfig.Labels["kompose.service.type"] == "NodePort" || composeServiceConfig.Labels["kompose.service.type"] == "LoadBalancer" {
+			log.Fatal("Failed to proceed as Nodeport or loadbalancer is defined without ports")
```

Comment:
@surajnarwade I think it should still convert.. Let's wait for @kadel 's opinion :+1: